.gitignore

```diff
@@ -51,6 +51,7 @@ phpcs.xml
 /bin/jtuber.sh
 /bin/tunnel.sh
 /bin/jt/
```

Comment:
Are these obsolete now?

Reply:
I'd call them legacy.
Technically, the older scripts location still works, and until everybody upgrades the scripts may still be in these locations.
So we'll remove them soon, but not yet.